source/blog/2016-05-25-modify-ifcfg-files.html.md

```diff
@@ -0,0 +1,74 @@
+title: Modifying oVirt-generated ifcfg files
```

Comment:
This is missing the frontmatter block starter...

Solution: add `---` as the first line.